File: C/lista2 - estrutura de repeticao/exercicio_5.c
#include <conio.h>
#include <stdio.h>

int main() {
	int age, ageGreaterThan50 = 0, personsAgeWithBetween10And20 = 0, heightBetween10And20 = 0, weightLessThan40 = 0, i;
	float height, averageHeight = 0, weight;
	
	for (i= 0; i< 25; i++) {
		printf("Informe uma idade: ");
		scanf("%d", &age);
		printf("Informe uma altura: ");
		scanf("%f", &height);
		printf("Informe um peso: ");
		scanf("%f", &weight);
		
		if (age > 50) {
			ageGreaterThan50++;
		} else if (age >= 10 && age <= 20) {
			personsAgeWithBetween10And20++;
			heightBetween10And20 += height;
		}
		
		if (weight < 40) {
			weightLessThan40++;
		}
	}
	
	averageHeight = (float) (heightBetween10And20 / personsAgeWithBetween10And20);
	
	printf("Total de pessoas com mais de 50 anos: %d\n", ageGreaterThan50);
	printf("Media da altura das pessoas com idade entre 10 e 20 anos: %f cm\n", averageHeight);
	printf("Percentual de pessoas com peso menor 40kg: %f", (float) (weightLessThan40 / (i) * 100));
}

File: Python/lista2 - estruturas de repeticao/exercicio_5.py
age_over_50 = 0
age_between_10_and_20 = 0
sum_height = 0
weight_less_than_40 = 0

for i in range(25):
    age = int(input('Informe sua idade: '))
    height = float(input('Informe sua altura em cm: '))
    weight = float(input('Informe seu peso: '))

    if age > 50:
        age_over_50 += 1
    elif age >= 10 and age <= 20:
        age_between_10_and_20 += 1
        sum_height += height

    if weight < 40:
        weight_less_than_40 += 1

print(f'Quantidade de pessoas com mais de 50 anos: {age_over_50}')
print(f'Média de altura das pessoas entre 10 e 20 anos: {sum_height / age_between_10_and_20}cm')
print(f'Porcentagem de pessoas com peso menor que 40kg: {(weight_less_than_40 / i) * 100}%')
